config_examples/config_freqai_futures.example.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
I think it would make more sense to provide freqAI configurations as partial samples.

they should only contain the `"freqai"` key - and can then be loaded via `add_config_files` in other parameters.
The freqAI part has no (meaningful) difference between spot and futures pairs.
instead, we should provide configs based on provided models (LightGBM, Catboost) - as they require slightly different `"model_training_parameters"`.